metadata/metadata.ts

```diff
@@ -7605,4 +7605,21 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
     image: `../public/erc1155/images/2032.png`,
     attributes: [{ trait_type: "Purpose", value: "Decoration" }],
   },
+  "Rice Panda": {
+    name: "Rice Panda",
+    description: "",
+
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image: "../public/erc1155/images/question_mark.png",
```

Comment:
Do we have an image and description for this?

We can use the `3x3_bg.png` file + the art for the panda to create the open sea image 👌